lib/wpscan/wp_target/wp_must_use_plugins.rb

```diff
@@ -10,7 +10,7 @@ def has_must_use_plugins?
 
       if response && [200, 401, 403].include?(response.code)
         hash = WebSite.page_hash(response)
-        return true if hash != error_404_hash && hash != homepage_hash
+        return true if hash != site_404_hash && hash != homepage_hash
```

Comment:
why don't you check against the hash set here?

Reply:
OK. I've changed it to use the existent method.